.gitignore

```diff
@@ -12,3 +12,6 @@ target
 
 # The Git repositories checked out by the integration test framework.
 integration-tests/.repos
+
+# Specific file exclusions.
```

Comment:
So it's an _inclusion_ in the repo ;). Perhaps we can just move this to line 7, so that a comment isn't necessary?